README.md

```diff
@@ -84,6 +105,15 @@ For example, this removes the gremlin icon in the gutter for non breakable space
 }
 ```
 
+To test this out, copy some of the [characters](https://unicode-table.com/en/blocks/latin-1-supplement/) or try the sample below.
```

Comment:
Should this be part of **Specifying a Range of Invalid Characters** instead of here inside **Hiding the gremlin icon in the gutter for a character**?

Reply:
Yes, typo. Should have been up in the prior section.